Header header
string lock_status
int8 battery_pct
string power_source
bool laser